PRODUCTIONISATION.md

```diff
@@ -49,8 +49,8 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 ### 3. Environments
 
 * 3.1 Application has corresponding environments and hosting for:
-  * [ ] 3.1.1 `testing` for internal delivery team
-  * [ ] 3.1.2 `staging` for showcasing to customer
+  * [ ] 3.1.1 `testing` for internal delivery team, this environment should only be accessible to engineers
+  * [ ] 3.1.2 `staging` for showcasing to customer, with restricted access and its setup should mirror production
```

Comment:
>setup should mirror production

All environments should have the same setup, though it is understandable to not operate staging/testing at the same scale as production (less/smaller infra is fine, for instance)

Reply:
See https://github.com/madetech/productionisation/pull/51/commits/323f30c34e0ebac807361540bbf598dcc86c3e66 for changes.